<!DOCTYPE html>
<html>
  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name=viewport content="width=device-width, initial-scale=1">
  
  <title>home | Weida Gong | weidagong.com</title>
  
  
  <meta name="description" content="Weida Gong">
  
  <meta name="keywords" content="jekyll, jekyll theme, jekyll framework, stuffthings">
  
    
  <link href="../css/main.css" media="screen, projection" rel="stylesheet" type="text/css">
  <link rel="canonical" href="">
  <link href="../favicon.ico" rel="icon">
  <link rel="alternate" type="application/atom+xml" title="Weida Gong | weidagong.com" href="/atom.xml"/>
  
  <!-- Social media content metadata -->


<meta property="twitter:card" content="summary" />
<meta property="twitter:site" content="" />
<meta property="twitter:url" content="" />
<meta property="twitter:title" content="home" />
<meta property="twitter:description" content="Rob Lampe" />
<meta property="twitter:creator" content="" />
<meta property="twitter:image:src" content="" />


  
 
</head>

  <body>
    <main class="wrap">
      
  <header role="banner">
      <a href=""><img src="../images/banner3-xl.jpg" srcset="/ 200w, ../images/banner3-med.jpg 641w, ../images/banner3-large.jpg 1025w, ../images/banner3-xl.jpg 1441w" alt="Weida Gong | wdgong.com banner image" title="Weida Gong | wdgong.com banner image" class="center"></a>
  </header>
  
  
      <nav class="navbar">
  <nav class="row">
    <div id="nav">
      <ul class="columns medium-8 left primary-nav">

	<li id="toggleMenu"><a href="#nav">Menu</a></li>
	<li id="hideMenu"><a href="#toggleMenu">Menu</a></li>
	
	
	  
		  
		  <li><a href="../index.html">home</a></li>
		    
	  
	
	  
		  <li class="active"><a href="research.html">research</a></li>
		
	
	  
		  
		  <li><a href="../publications/publications_.html">publications</a></li>
		    
	  
	
	  
		  
		  <li><a href="../cv/cv.html">cv</a></li>
		
	
	  
		  
		  <li><a href="../links/links.html">links</a></li>
		    
	  
	
      </ul>

      <ul class="columns medium-4 right primary-nav">
    <!--
	
	  <li class="right"><a href="https://facebook.com/robhlampe"><span class="icon-facebook socials"></span></a></li>
	
	  <li class="right"><a href="https://github.com/rhlampe"><span class="icon-github socials"></span></a></li>
	
	  <li class="right"><a href="https://www.linkedin.com/in/robhlampe"><span class="icon-linkedin socials"></span></a></li>
	
    -->
      </ul>
    </div>
  </nav>
</nav>

      <article class="row">
  <section class="small-12 large-8 columns page-content">

<h2>Current Research</h2>
<ul>

<li><img src="../images/NRE.png" align="left" width="300" height="150" hspace="20" style="box-shadow: 3px 3px 1px #888888; border:3px solid black; margin-bottom:20px"> 
<b>Molecular Insights into a Dinoflagellate Bloom</b><br /><br />

<p align="justify">Estuaries are highly dynamic and productive ecosystems, characterized by significant temporal and spatial gradients in both biotic and abiotic factors. As primary producers, phytoplankton play an important role in estuaries, thus a clear understanding of how phytoplankton respond to spatiotemporal environmental gradients is necessary to ensuring healthy and sustainable ecosystems. Our research focuses on elucidating the interactions between biotic and abiotic factors through combining plankton molecular physiology with more traditional water quality measurements to reveal the fundamental cellular processes that guide these interactions. Using a combination of transcriptomic sequencing and network analysis, we detected distinct inferred phytoplankton physiologies within the Neuse River Estuary (NRE), with phytoplankton allocating more resources to cellular component synthesis and carbon metabolism in the upper estuary, and to nutrient metabolisms and transporters in the lower estuary, consistent with gradients in nitrogen availability. Nutrient concentrations were shown to have positive correlation with cellular component synthesis and metabolic activities, and negative correlation with nutrient metabolisms and transporters. The relationship between phytoplankton metabolic functions as inferred through metatranscriptomic sequencing and environmental factors lays a solid foundation for examining how phytoplankton are responding to environmental changes, and is thus a valuable tool for marine ecosystem modeling and algal bloom prediction, which can be applied not just in the NRE, but to coastal waters worldwide.</p>

<p>Funded by North Carolina Sea Grant</p></li>

<li><b>Comparison between 18s, transcriptomic and mophological taxonomic identification</b><br></li><br /><br /><br /><br />


<h2>Past Research</h2>

<img src="../images/F3.png" align="left" width="150" height="300" hspace="20" style="box-shadow: 3px 3px 1px #888888; border:3px solid black"> 
<b>Molecular Insights into a Dinoflagellate Bloom</b><br /><br />

<p align="justify">In coastal waters worldwide, an increase in frequency and intensity of algal blooms has been attributed to eutrophication, with further increases predicted due to climate change. Yet, the cellular-level changes that occur in blooming algae remain largely unknown. In this research, comparative metatranscriptomics was used to investigate the underlying molecular mechanisms associated with a dinoflagellate bloom in a eutrophied estuary. We found that under bloom conditions, there is increased expression of metabolic pathways indicative of rapidly growing cells, including energy production, carbon metabolism, transporters and synthesis of cellular membrane components. In addition, there is a prominence of highly expressed genes involved in synthesis of membrane-associated molecules, including those for the production of glycosaminoglycans (GAGs), which may serve roles in nutrient acquisition and/or cell surface adhesion. Biotin and thiamine synthesis genes also increased expression along with several cobalamin biosynthesis-associated genes, suggesting processing of B12 intermediates by dinoflagellates. The patterns in gene expression observed are consistent with bloom-forming dinoflagellates eliciting a cellular response to elevated nutrient demands and to promote interactions with their surrounding bacterial consortia possibly in an effort to cultivate for enhancement of vitamin and nutrient exchanges and/or direct consumption. Our findings provide potential molecular targets for bloom characterization and management efforts.</p>

<p>Funded by North Carolina Sea Grant</p>

  </section>
</article>

    </main>
      <footer class="site-footer">
  <div class="row">
    <div class="small-6 columns">
    </div>
    <div class="small-6 columns">
    </div>
  </div>
</footer> 

  </body>
</html>